.gitignore

```diff
@@ -3,6 +3,7 @@
 /public/build
 /public/hot
 /public/storage
+/public/assets/images/*
```

Comment:
Ooooooh, you will hear me in the session telling them to add this to .gitignore just like Engy😂

Reply:
thank you very much